server/config/config.go

```diff
@@ -1070,7 +1071,10 @@ type PDServerConfig struct {
 	// There are some values supported: "auto", "none", or a specific address, default: "auto"
 	DashboardAddress string `toml:"dashboard-address" json:"dashboard-address"`
 	// TraceRegionFlow the option to update flow information of regions
+	// TODO: deprecate
```

Comment:
Can we directly mark it as deprecated?

Reply:
some users already relies on it, I need to do some compatibility processing. will do it in another PR.

Reply:
test